scripts/allowlist_utils.js

```diff
@@ -36,33 +36,48 @@ function isInitialAdminAccountsAvailable() {
     return process.env.INITIAL_ADMIN_ACCOUNTS;
 }
 
-function isInitialWhitelistedAccountsAvailable() {
-    return process.env.INITIAL_WHITELISTED_ACCOUNTS;
+function isInitialAllowlistedAccountsAvailable() {
+    if (process.env.INITIAL_ALLOWLISTED_ACCOUNTS) {
+        return process.env.INITIAL_ALLOWLISTED_ACCOUNTS;
+    }
+    if (process.env.INITIAL_WHITELISTED_ACCOUNTS) {
+        console.log("INITIAL_WHITELISTED_ACCOUNTS has been deprecated. Please use INITIAL_ALLOWLISTED_ACCOUNTS instead.");
```

Comment:
NIT: maybe we want to make this a warning? `console.warn`

Reply:
done